__tests__/app.test.js

```diff
@@ -0,0 +1,189 @@
+              "https://images.pexels.com/photos/279321/pexels-photo-279321.jpeg?w=700&h=700",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 10,
+            designer: "Wolfgang Warsch",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "That's just what an evil person would say!",
+            review_id: 11,
+            category: "social deduction",
+            review_img_url:
+              "https://images.pexels.com/photos/220057/pexels-photo-220057.jpeg?w=700&h=700",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 8,
+            designer: "Fiona Lohoar",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "One Night Ultimate Werewolf",
+            review_id: 8,
+            category: "social deduction",
+            review_img_url:
+              "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+            designer: "Akihisa Okui",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "Agricola",
+            review_id: 1,
+            category: "euro game",
+            review_img_url:
+              "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+            created_at: "2021-01-18T10:00:20.514Z",
+            votes: 1,
+            designer: "Uwe Rosenberg",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "Proident tempor et.",
+            review_id: 5,
+            category: "social deduction",
+            review_img_url:
+              "https://images.pexels.com/photos/209728/pexels-photo-209728.jpeg?w=700&h=700",
+            created_at: "2021-01-07T09:06:08.077Z",
+            votes: 5,
+            designer: "Seymour Buttz",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "Occaecat consequat officia in quis commodo.",
+            review_id: 6,
+            category: "social deduction",
+            review_img_url:
+              "https://images.pexels.com/photos/207924/pexels-photo-207924.jpeg?w=700&h=700",
+            created_at: "2020-09-13T14:19:28.077Z",
+            votes: 8,
+            designer: "Ollie Tabooger",
+            comment_count: "0",
+          },
+          {
+            owner: "mallionaire",
+            title: "Settlers of Catan: Don't Settle For Less",
+            review_id: 13,
+            category: "social deduction",
+            review_img_url:
+              "https://images.pexels.com/photos/1153929/pexels-photo-1153929.jpeg?w=700&h=700",
+            created_at: "1970-01-10T02:08:38.400Z",
+            votes: 16,
+            designer: "Klaus Teuber",
+            comment_count: "0",
```

Comment:
we should be expecting comment_count back as a number rather than a string, this can be changed to send back a number within the model!